app/stores/ada/AdaWalletsStore.js

```diff
@@ -160,6 +167,34 @@ export default class AdaWalletsStore extends WalletStore {
     this.refreshWalletsData();
   };
 
+  // =================== NOTIFICATION ==================== //
+  showWalletCreatedNotification = (): void => {
+    const notification: Notification = {
+      id: globalMessages.walletCreatedNotificationMessage.id,
+      message: globalMessages.walletCreatedNotificationMessage,
+      duration: config.wallets.WALLET_CREATION_SUCCESS_NOTIFICATION_DURATION,
+    };
+    this.actions.notifications.open.trigger(notification);
+  }
+
+  showWalletRestoredNotification = (): void => {
+    const notification: Notification = {
+      id: globalMessages.walletRestoredNotificationMessage.id,
+      message: globalMessages.walletRestoredNotificationMessage,
+      duration: config.wallets.WALLET_CREATION_SUCCESS_NOTIFICATION_DURATION,
```

Comment:
`WALLET_CREATION_SUCCESS_NOTIFICATION_DURATION` the name seems a little off. we should create a new one or if we are going to reuse the same, we should have a more generic name.

Reply:
It depends upon how you think.

![image](https://user-images.githubusercontent.com/19986226/50274404-8712ec00-0480-11e9-8dc0-203cf155ec33.png)


My way of thinking is we are on `ADD WALLET` page and we have 3 ways of Wallet creation.
End result will be a new wallet created, and we are showing notification for new wallet created.

I can't find more generic name, will make one constants for each way of wallet creation.
So we will have 3 constants with same value.